qcodes/tests/drivers/test_keithley_26xx.py

```diff
@@ -41,8 +43,8 @@ def test_smu_channels_and_their_parameters(driver):
         assert 'current' == smu.mode()
         smu.mode('voltage')
 
-        assert 'off' == smu.output()
-        smu.output('on')
+        assert 0 == smu.output()
+        smu.output(1)
```

Comment:
@astafan8 is right, but we usually adhere to the `assert actual_value == expected_value` way of asserting in tests. So that would be `assert smu.output() == 1`.

This is a long standing war between Mikhail and me 😄

Reply:
yes yes :) in this case i followed the format of the already existing `assert`ions